library/src/main/java/com/google/android/exoplayer2/extractor/mp4/FragmentedMp4Extractor.java

```diff
@@ -1192,7 +1192,7 @@ private static DrmInitData getDrmInitDataFromAtoms(List<Atom.LeafAtom> leafChild
         if (uuid == null) {
           Log.w(TAG, "Skipped pssh atom (failed to extract uuid)");
         } else {
-          schemeDatas.add(new SchemeData(uuid, MimeTypes.VIDEO_MP4, psshData));
+          schemeDatas.add(new SchemeData(uuid, MimeTypes.CENC, psshData));
```

Comment:
Thank for your replay.

It accept "video / mp3" only on Android 7.1.1  or higher.
https://android.googlesource.com/platform/frameworks/av/+/01a977aa75c296c5467024a7beac50cbd4ed0335%5E%21/#F4

Should I change the definition class?